### PR TITLE
Feat: begun translatin to uzbek lang

### PR DESCRIPTION
**Pull request by Zukhrik**
### Conventions
- [ ] Please check your messages [against the guidelines](https://cbea.ms/git-commit/) if not, perform [interactive rebase](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history)
- [ ] [Link an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) your pull request closes or relates

### CODE REVIEW

**Review by zerobias — APPROVED** [review]
Amazing, thanks 🔥

File: beta/src/languages.ts [hunk 1]
```diff
@@ -52,119 +52,134 @@ export const translations = {
     text: {
       en: "This page is not translated yet",
       ru: "Эта страница еще не переведена",
+      uz: "Bu sahifani hali tarjimasi yo'q"
```
Comment (45m0d3u5) [inline beta/src/languages.ts:55]: Bu sahifa hali tarjima qilinmagan

File: beta/src/languages.ts [hunk 2]
```diff
@@ -52,119 +52,134 @@ export const translations = {
...
       en: "On this page",
       ru: "Оглавление",
+      uz: "Mundarija"
     },
   },
   EditThisPage: {
     text: {
       en: "Edit this page",
       ru: "Внести правки",
+      uz: "Sahifani tahrirlash"
```
Comment (45m0d3u5) [inline beta/src/languages.ts:113]: O'zgartirish kiritish

File: beta/src/navigation.ts [hunk 3]
```diff
@@ -5,65 +5,65 @@ import { getTextLocalized, createLink, type LText } from "./languages";
...
         link: "/introduction/community",
       },
       {
-        text: { en: "Installation", ru: "Установка" },
+        text: { en: "Installation", ru: "Установка", uz: "O'rnatma" },
         link: "/introduction/installation",
       },
       {
-        text: { en: "Ecosystem", ru: "Экосистема effector" },
+        text: { en: "Ecosystem", ru: "Экосистема effector", uz: "Effector ekosistema" },
```
Comment (45m0d3u5) [inline beta/src/navigation.ts:23]: Effector ekosistemasi

File: beta/src/navigation.ts [hunk 4]
```diff
@@ -101,22 +102,22 @@ const defaultSidebar: LSidebarGroup[] = [
     ],
   },
   {
-    text: { en: "Explanation", ru: "Погружение" },
+    text: { en: "Explanation", ru: "Погружение", uz: "Tushuntirish" },
     items: [
       {
-        text: { en: "Glossary", ru: "Глоссарий" },
+        text: { en: "Glossary", ru: "Глоссарий", uz: "Glossary" },
```
Comment (45m0d3u5) [inline beta/src/navigation.ts:108]: Lug'at

File: beta/src/navigation.ts [hunk 5]
```diff
@@ -207,55 +208,55 @@ const effectorReact = [
...
       {
-        text: { en: "Provider" },
+        text: { en: "Provider", uz: "Provayderlar" },
         link: "/api/effector-react/Provider",
       },
     ],
   },
   {
-    text: { en: "Gates" },
+    text: { en: "Gates", uz: "Geyts" },
```
Comment (45m0d3u5) [inline beta/src/navigation.ts:242]: Здесь лучше оставить Gates

File: beta/src/navigation.ts [hunk 6]
```diff
@@ -207,55 +208,55 @@ const effectorReact = [
...
     items: [
       {
-        text: { en: "Gate" },
+        text: { en: "Gate", uz: "Geyts" },
```
Comment (45m0d3u5) [inline beta/src/navigation.ts:245]: Здесь лучше оставить Gates

Comment (45m0d3u5) [reply beta/src/navigation.ts:245]: Gate*

**Pull request merged**